packages/@okta/vuepress-site/docs/guides/request-user-consent/overview/index.md

```diff
@@ -4,164 +4,19 @@ title: Request user consent during authentication
-
-    ```bash
-    curl -v -X GET \
-    -H "Accept: application/json" \
-    -H "Content-Type: application/json" \
-    -H "Authorization: SSWS ${api_token} \
-    "https://${yourOktaDomain}/api/v1/users/${userId}/grants"
-    ```
-
-    The response should contain the grant you created when you clicked **Allow** in the previous step.
-
-    ```json
-    [
-        {
-            "id": "oag4xfx62r6S53kHr0h6",
-            "status": "ACTIVE",
-            "created": "2018-04-23T21:53:25.000Z",
-            "lastUpdated": "2018-04-23T21:53:25.000Z",
-            "issuerId": "auscl1o4tnf48w5Wt0h7",
-            "issuer": null,
-            "clientId": "xfnIflwIn2TkbpNBs6JQ",
-            "userId": "00u5t60iloOHN9pBi0h7",
-            "scopeId": "scpcl1o4toFjganq10h7",
-            "_links": {
-                "app": {
-                    "href": "https://${yourOktaDomain}/api/v1/apps/0oaaggpxeqxTDuP780h7",
-                    "title": "Acme OIDC Client"
-                },
-                "authorizationServer": {
-                    "href": "https://${yourOktaDomain}/api/v1/authorizationServers/auscl1o4tnf48w5Wt0h7",
-                    "title": "My Authorization Server"
-                },
-                "scope": {
-                    "href": "https://${yourOktaDomain}/api/v1/authorizationServers/auscl1o4tnf48w5Wt0h7/scopes/scpcl1o4toFjganq10h7",
-                    "title": "openid"
-                },
-                "self": {
-                    "href": "https://${yourOktaDomain}/api/v1/users/00u5t60iloOHN9pBi0h7/grants/oag4xfx62r6S53kHr0h6"
-                },
-                "revoke": {
-                    "href": "https://${yourOktaDomain}/api/v1/users/00u5t60iloOHN9pBi0h7/grants/oag4xfx62r6S53kHr0h6",
-                    "hints": {
-                        "allow": [
-                            "DELETE"
-                        ]
-                    }
-                },
-                "client": {
-                    "href": "https://${yourOktaDomain}/oauth2/v1/clients/xfnIflwIn2TkbpNBs6JQ",
-                    "title": "Acme OIDC Client"
-                },
-                "user": {
-                    "href": "https://${yourOktaDomain}/api/v1/users/00u5t60iloOHN9pBi0h7",
-                    "title": "Saml Jackson "
-                },
-                "issuer": {
-                    "href": "https://${yourOktaDomain}/api/v1/authorizationServers/auscl1o4tnf48w5Wt0h7",
-                    "title": "My Authentication Server"
-                }
-            }
-        }
-    ]
-    ```
-
-## Troubleshooting
+When an application needs to get a new access token from an authorization server, it may not need to prompt the user for consent if they have already consented to the specified scopes. Consent grants remain valid until the user or admin manually revokes them, or until the user, application, authorization server, or scope is deactivated or deleted.
```

Comment:
NIT: When an application needs to get a new access token from an authorization server, it will not prompt the user for consent if they have already consented to the specified scopes.